dist/templates/ovnkube-db.yaml.j2

```diff
@@ -89,10 +89,6 @@ spec:
           name: host-var-log-ovs
         - mountPath: /var/log/ovn/
           name: host-var-log-ovs
-        # for the iptables wrapper
```

Comment:
local gateway mode still runs iptables binaries from ovnkube-node, so we still need this stuff.

maybe add a note to the README or something clarifying that the deployer needs to make sure geneve (and maybe VXLAN) ports are open between nodes?

otherwise lgtm

Reply:
@danwinship I have updated FAQ.md to include what ports to open on each of the nodes in k8s cluster.

Also, i have removed the iptables wrapper scripts only for the DB pods and not for the ovnkube-node Pod where we still use `iptables` for both local and shared gateway mode.

Reply:
oh, somehow I saw "ovnkube-db" and read it as "ovnkube-node"...